source/applicability/basic_structure.rst

```diff
@@ -1607,17 +1607,14 @@ communication remains established and to detect any communication
 disruptions between site and supervision system. For any subsystem
 alarms are used instead.
 
-The secondary purpose of watchdog messages is to provide a timestamp that can
-be used for simple time synchronization.
-
-* Time synchronization using the watchdog message should be configurable at the
-  site (enabled/disabled)
-* If time synchronization is enabled, the site should synchronize its clock
-  using the timestamp from watchdog messages – at communication establishment and
-  then at least once every 24 hours.
-* The interval duration for sending watchdog messages should be
-  configurable at both the site and the supervision system. The default
-  setting should be (1) once a minute.
+The secondary purpose of watchdog messages is to provide a timestamp
+that can be used to check time synchronization. However watchdog
+messages should not be used to adjust the clock. Instead
+NTP (Network Time Protocol) should be used to synchronize clocks.
```

Comment:
Do we need to be specific about adding NTP as a requirement? Many traffic controller can use other time synchronization methods, such as GPS or similar. There is also high precision alternatives such as PTP.

I suggest writing "Instead, a high precision time synchronization method should be used (e.g. NTP or similar)." or simply leaving this sentence out.